### PR TITLE
Ms updates

### PR DESCRIPTION
**Pull request by manijshrestha**
Added DaggerAwareActivity and BaseNavigationActivity to support multi activity scenario.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -59,6 +59,7 @@ android {
     }
 
     aaptOptions {
+        //Note: If you did not apply PNG optimization on your own, you may want this to be true
```
Comment (manjunathc23) [inline app/build.gradle:62]: 👍

File: app/src/main/java/zeta/android/apps/activities/BaseNavigationActivity.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+public abstract class BaseNavigationActivity extends DaggerAwareActivity implements INavigationFragmentManager,
+        NavigationView.OnNavigationItemSelectedListener {
+
+    protected NavigationFragmentManager mNavigationFragmentManager;
+
+    @SuppressWarnings("unused")
+    @Inject
+    protected void setNavigationFragmentManager(NavigationFragmentManager navigationFragmentManager) {
+        mNavigationFragmentManager = navigationFragmentManager;
+    }
```
Comment (manjunathc23) [inline app/src/main/java/zeta/android/apps/activities/BaseNavigationActivity.java:19]: 👍

File: app/src/main/java/zeta/android/apps/component/NavigationActivityComponent.java [hunk 3]
```diff
@@ -3,9 +3,12 @@
...
 import zeta.android.apps.activities.NavigationActivity;
 import zeta.android.apps.dagger.ActivityScope;
+import zeta.android.apps.modules.ActivityModule;
 
 @ActivityScope
-@Subcomponent
+@Subcomponent(modules = {
+        ActivityModule.class
+})
 public interface NavigationActivityComponent {
```
Comment (manjunathc23) [inline app/src/main/java/zeta/android/apps/component/NavigationActivityComponent.java:12]: 👍

**Comment by manjunathc23** [comment]
@manijshrestha : Merge whenever you are ready! Looks neat to me.

**Comment by manijshrestha** [comment]
🆒 Thanks, I will merge this now. It will go in whenever we do next release.

**Pull request merged**